docs/ides/gateway.md

```diff
@@ -74,3 +74,44 @@ explaining this IDE specification.
    ![Gateway IDE Opened](../images/gateway/gateway-intellij-opened.png)
 
 > Note the JetBrains IDE is remotely installed into `~/.cache/JetBrains/RemoteDev/dist`
+
+## Configuring Gateway to use self-signed certificates
+
+When attempting to connect to a Coder deployment that uses self-signed certificates,
+you may receive the following error in Gateway:
+
+```console
+Failed to configure connection to https://coder.internal.enterprise/: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
+```
+
+To resolve this issue, you will need to add Coder's certificate to the Java trust store
+present on your local machine. Here is the default location of the trust store for
+each OS:
+
+```console
+# Linux
+$JAVA_HOME/lib/security/cacerts
+/etc/pki/java/cacerts
+/etc/ssl/certs/java/cacerts
+
+# macOS
+$(/usr/libexec/java_home)/lib/security/cacerts
+$(/usr/libexec/java_home)/jre/lib/security/cacerts
+
+# Windows
+C:\Program Files (x86)\Java\jre<version>\lib\security\cacerts
+```
```

Comment:
It seems in practice Gateway always uses the `cacerts` bundle that ships with it rather than any global trust store (at least so far that has been the case) so I think we should instead say the location is `<Gateway installation directory>/jbr/lib/security/cacerts`.

I might have some of these paths wrong anyway.  I think some are for the key store only and the trust store default can also use a file called `jssecacerts` but I am not 100% sure.

Reply:
thanks @code-asher - let me know if my recent push is correct.

Reply:
Oops sorry my example was super vague.  It would be the `<Gateway installation directory>/jbr/lib/security/cacerts` path verbatim for every OS.

So something like:

```
To resolve this issue, you will need to add Coder's certificate to the Java trust store
present on your local machine. The trust store used by Gateway comes bundled
in the installation and can be found at 
`<Gateway installation directory>/jbr/lib/security/cacerts`.
```

Or maybe:

```
...can be found at `jbr/lib/security/cacerts` from the Gateway installation directory.
```